### PR TITLE
QPPCT-441: test coverage

### PR DESCRIPTION
**Pull request by clydet**
### Information
- QPPCT-441

### Changes proposed in this PR:
- Increased test coverage

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/Converter.java [hunk 1]
```diff
@@ -270,7 +270,8 @@ public void setReportDetails(AllErrors details) {
 		 * @return input stream of QPP validation details
 		 */
 		public InputStream streamRawValidationDetails() {
-			return new ByteArrayInputStream(qppValidationDetails.getBytes(Charset.defaultCharset()));
+			String raw = (qppValidationDetails != null) ? qppValidationDetails : "";
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/Converter.java]: MAJOR Convert this usage of the ternary operator to an "if"/"else" structure. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1774)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by omnivion** [comment]
I think new tests should be written using junit 5.

File: converter/src/test/java/gov/cms/qpp/conversion/model/error/AllErrorsTest.java [hunk 2]
```diff
@@ -53,4 +55,11 @@ public void testToString() {
 		assertWithMessage("Must contain formatted string")
 				.that(objectUnderTest.toString()).contains(error.toString());
 	}
+
+	@Test
+	public void testArgConstructor() {
+		List<Error> errors = new ArrayList<>();
+		errors.add(new Error());
+		new AllErrors(errors);
```
Comment (halprin) [inline converter/src/test/java/gov/cms/qpp/conversion/model/error/AllErrorsTest.java]: I would call `AllErrors#getErrors` afterward to verify that it is the same as the constructor argument `errors`.

**Comment by saquino0827** [comment]
The PR looks fine and the JUnit 5 conversion can be handled separately.

**Pull request merged**